### PR TITLE
 Update elm surfdata_map and IC files for 1950 compsets

### PR DESCRIPTION
**Pull request by wlin7**
1950 compsets for ne30pg2 are currently using surfdata_ne30pg2_simyr2000_c190408.nc
and do not specify an finidat file. This PR sets to use the simyr1950
surfdata map and specify an IC file generated by land-only spin-up with
CRUNCEPv7 forcing cycling between 1941 and 1960.

[BFB] except for 1950 compsets

### CODE REVIEW

File: components/elm/cime_config/config_component.xml [hunk 1]
```diff
@@ -74,8 +74,7 @@
       <value compset="20TR.*_EAM%CMIP6_ELM">20thC_CMIP6_transient</value>
       <value compset="20TR(?:SOI)?_EAM%CMIP6_ELM*_BGC%B">20thC_CMIP6bgc_transient</value> 
       <value compset="20TR.*_ELM%[^_]*BGC">20thC_bgc_transient</value>
-      <value compset="1950(?:SOI)?_EAM%CMIP6-LR.*_ELM">1950_CMIP6LR_control</value>
-      <value compset="1950(?:SOI)?_EAM%CMIP6-HR_ELM">1950_CMIP6HR_control</value>
```
Comment (bishtgautam) [inline components/elm/cime_config/config_component.xml:78]: Is the support for `CMIP6-LR_ELM` and `CMIP6-HR_ELM` being discontinued?

Comment (wlin7) [reply components/elm/cime_config/config_component.xml:78]: Thanks for the careful review, @bishtgautam . Yes, they are discontinued. CMIP6-LR and CMIP6-HR only relevant for V1 code base.

**Review by bishtgautam — APPROVED** [review]

**Comment by wlin7** [comment]
Merged to next.

**Comment by wlin7** [comment]
Merged to master.

**Pull request merged**